domains/uiopler.json

```diff
@@ -0,0 +1,11 @@
+{
+  "description": "uiopler's website",
+  "owner": {
+    "username": "uiopler",
+    "email": "uiopler.dev@gmail.com",
+    "twitter": "uiopler_dev"
+  },
+  "record": {
+    "CNAME": "uiopler.netlify.app"
```

Comment:
Netlify doesn't work with a CNAME at the moment, and neither does Vercel. As an alternative, there is [Railway](https://railway.app?referralCode=jj), and [Deta](https://www.deta.sh/)